nlp_primitives/lsa.py

```diff
@@ -80,3 +87,25 @@ def lsa(array):
             return pd.Series(arr)
 
         return lsa
+
+
+def _validate_trainer(trainer):
+    if not isinstance(trainer, Pipeline):
+        raise ValueError('Given LSA pipeline is not a Pipeline object')
+    if len(trainer.steps) != 2:
+        raise ValueError('Given LSA pipeline contains an incorrect number of transformers')
+    if not isinstance(trainer.steps[0][1], TfidfVectorizer):
+        raise ValueError('Given LSA pipeline does not have a TfIdfVectorizer as its first transformer')
+    if not isinstance(trainer.steps[1][1], TruncatedSVD):
+        raise ValueError('Given LSA pipeline does not have a TruncatedSVD as its second transformer')
+    try:
+        trainer.steps[0][1].vocabulary_
+        trainer.steps[1][1].components_
+    except AttributeError:
+        raise ValueError('Given LSA pipeline should be pre-fitted')
+
+
+def make_trainer(corpus, random_state=42):
```

Comment:
I think it would be really useful to see an example of using this in the docstring (either for this function, the LSA docstring, or both). Also, I think this function should probably be added to the `__init.py__` file so that users can access it through the package.